README.rst

```diff
@@ -81,6 +81,11 @@ and then install the dependencies separately::
 and install Neuron using the traditional package installer available here
 `https://neuron.yale.edu/neuron/ <https://neuron.yale.edu/neuron/>`_.
 
+As a final step, .mod files must be compiled with Neuron by running:
+
+   $ python -c 'import site; print(site.getsitepackages()[1] + \hnn_core\)' | cd
```

Comment:
This line feels too hacky. I wouldn't want users to edit the anaconda install. When you run setup, it first tries to build and then copies the built files over

it's possible that [Manifest.in needs to be updated for Windows](https://github.com/jonescompneurolab/hnn-core/blob/master/MANIFEST.in#L8) because that tells what files to copy